pylzy/tests/test_data/lzy_test_project/LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2016 The Python Packaging Authority (PyPA)
```

Comment:
Why we need license here?

Reply:
Because I copypasted this boilerplate from pypa repository.
And this license requires to include this license in this case at first glance.